media/css/firefox/features/article.scss

```diff
@@ -9,6 +9,31 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/components/callout';
 @import '~@mozilla-protocol/core/protocol/css/components/breadcrumb';
 
+.mzp-c-breadcrumb {
+    padding-top: $spacing-md;
+    padding-bottom: $spacing-md;
```

Comment:
**Suggestion:** This change would probably be better placed in the actual Protocol design system rather than it just being in Bedrock?

Reply:
I've come to understand that any updates regarding nav (and breadcrumbs) in Protocol are now paused in expectations of a bigger redesign coming up, so probably if something of this is supposed to end up upstream, it's gonna be in the next design language anyways — and so with the nav due redesigning in a few months, I see this whole improvement as a tentative fix nonetheless, to be completely replaced with something more systematic, come summer(?)

For now I've managed to align the subnav & breadcrumbs perfectly to the main menu. But not sure I really love the outcome on some pages. 🤷

I also don't fancy the way I've patched up the overrides to implicit breadcrumb protocol styles here locally to with subnav 🤷

Reply:
@reemhamz Maybe the whole fix should be upstream https://github.com/mozilla/protocol/issues/933 after all — the ideal combo might be: subnav stays, only breadcrumbs get fixed (e.g. https://github.com/mozilla/bedrock/compare/main...8390523 but upstream instead), and what needs to change to better align with the rest is the main nav, not the other subnav elements (which are properly aligned to content in all breakpoints; it's the masthead that gets narrower sometimes…)

Reply:
Looking at your changes, I actually agree that the breadcrumb component should get fixed upstream in Protocol rather than have it override itself within Bedrock. However, I don't quite understand why the main nav needs to be changed? Could you further elaborate on that?

Reply:
> I don't quite understand why the main nav needs to be changed? Could you further elaborate on that?

I'll post more info to the original issue #14081 to explain it a bit more, but just quickly following up your investigation from https://github.com/mozilla/bedrock/issues/14081#issuecomment-2054771820:

> I think I came to understand that the reason there's considered to be a misalignment is actually because the Breadcrumb component is aligned similarly to the subnav that we have. Now the question is, do we want to:
> 1. Change the alignments of both subnav and Breadcrumb component to align better with the main nav of the site
> 2. Only change the Breadcrumb component alignment
> 3. Keep things as is

Or there is 4.: Keep breadcrumbs and subnav aligned to mzp-l-content as it is now, but overcome the perception of these being misaligned in comparison to the main nav by actually tweaking some mq breakpoint styling of the nav instead, bc it's the main masthead that doesn't match mzp-l-content alignment in a handful of breakpoints. But I'll explain that a tad better with videos & examples so I'll keep you posted there…

TL;DR 3.) is out of the equation and https://github.com/mozilla/protocol/issues/933 needs fixing anyways, so 2.) will happen upstream https://github.com/mozilla/protocol/pull/938; 1.) was this PR and I don't like the result, so I will see about 4.) eventually, probably again in Protocol itself, but that's a separate issue then.